### ISSUE TITLE
input data added with --file is not loaded into container executor

### ISSUE THREAD

**Issue by esebrei** [kubeshop/testkube#2950]
**Describe the bug**

creating a test and adding input data with `--file` to a test does not add the file content to the file `/data/test-content`. 

In fact, the file `/data/test-content` is missing.

**To Reproduce**
1. Run `testkube create test --name container-test --type [your-custom-executor-type] --file test-content.txt`
2. Use `ENTRYPOINT|CMD` and `find .` to display all files and folders in the container
3. Run `testkube run test container-test`
4. `/data/test-content` is missing in log output:

```
[...]
./srv
./data
./.dockerenv
[...]
```

**Expected behavior**

The doc https://kubeshop.github.io/testkube/test-types/container-executor#input-data states that the `--file` should add a file `/data/test-content` with the content of the passed file to the container:

```
[...]
./srv
./data
./data/test-content
./.dockerenv
[...]
```

**Version / Cluster**
- Which testkube version? 
`1.7.29` and `1.8.7`
(working with `1.7.28`)

**Comment by vsukhin** [comment]
Thank you @esebrei We will check it out

**Comment by vLia** [comment]
Hello @esebrei,

Depending on the base image you are using, `WORKDIR` will likely be different than root (`/`). The `/data/test-content` file might not be located under the current working directory (`.`) you are checking your files in.

Alternatively, you could use [`--copy-files`](https://kubeshop.github.io/testkube/using-testkube/tests/tests-creating/#mapping-local-files) to have more control over the file names mapped into the container.

**Comment by esebrei** [comment]
The base image should not be the problem, as I did not change the version (`python:3.9-slim`) and it did work with `1.7.28`.

But I did update the version of testkube  CLI. The file `test-content` cannot be found anywhere in the docker image. I run the `find` command in `/`.

`V1.7.28`:

`--file test.py`  adds the content of the file to `/data/test-content`.

`V1.7.29`:

`--file test.py`  **does not** add the content of the file to `/data/test-content`.
